### PR TITLE
Fixed #25851 -- Updated link to deprecated assignment tag docs.

### PR DESCRIPTION
**Pull request by alasdairnicol**

### CODE REVIEW

File: docs/howto/custom-template-tags.txt [hunk 1]
```diff
@@ -499,6 +499,8 @@ positional arguments. For example:
 
     {% my_tag 123 "abcd" book.title warning=message|lower profile=user.profile %}
 
+.. _howto-custom-template-tags-simple-tag-assignment:
```
Comment (timgraham) [inline docs/howto/custom-template-tags.txt]: I think it's confusing to put links to places in the docs that don't have a heading.
We can remove the ".. _howto-custom-template-tags-assignment-tags:" too and the remaining usage in the 1.4 release notes.

Comment (alasdairnicol) [inline docs/howto/custom-template-tags.txt]: > I think it's confusing to put links to places in the docs that don't have a heading.
Understood. I could link to the simple tags heading, but it's quite a long section. Do you think that's ok, or is it possible to create a subheading?

> We can remove the ".. _howto-custom-template-tags-assignment-tags:" too and the remaining usage in the 1.4 release notes.
OK, I will do this on Thursday.

**Comment by alasdairnicol** [comment]
@timgraham I've updated the pull request with your suggestions.

**Comment by timgraham** [comment]
merged in b6dd0afead80a17d0ac8c3ba35c510afac32a0b8, thanks!

**Pull request closed without merging**